app/views/users/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<ul>
+  <% @user.each do |user| %>
+    <li>
+      <%= user.id %>
+    </li>
```

Comment:
Ha everything don't look good to me! You copied the setup of the other view without thinking about it.

Think about what the @user object is here.